ocfweb/tv/templatetags/tv_formatting.py

```diff
@@ -0,0 +1,14 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def tv_lab_hours(hour):
+    if hour:
+        if hour.open.minute != 0 or hour.close.minute != 0:
+            fmt = '{:%-I:%M%P}-{:%-I%M%P}'
+        else:
+            fmt = '{:%-I%P}-{:%-I%P}'
```

Comment:
Yeah I figured white-space after looking it up but for some reason text-wrap seems to work for me in Chrome (even though it's not an implemented property?) while white-space: nowrap just truncates the text. I'm going to explore this a bit more.